health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/CampaignBoundary.js

```
@@ -0,0 +1,520 @@
+            {bHierarchy.length > 1 && bHierarchy.slice(1, -1).map((item, ind) => {
+                //item-Country,Province,Locality,District,Village
+                if (parent_group[item] && Array.isArray(parent_group[item])) {
+                    return parent_group[item].map((item1, idx) => {
+                        //item1-Province-[Sinoe],District-[Jedepo,Jeade]
+
+
+                        console.log("status", ind + 2, boundaryStatus[ind + 2])
+
+                        return Array.isArray(item1) && (!boundaryStatus[ind + 2]) ? (
+                            //make a super-comp that contains SubBoundary View
+                            <Card>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)